src/api/baseapi.cpp

```diff
@@ -1148,14 +1148,28 @@ bool TessBaseAPI::ProcessPagesInternal(const char* filename,
       return false;
     } else {
       CURLcode curlcode;
+      auto error = [curl, curlcode](const char* function) {
```

Comment:
Capture `curlcode` by ref? Otherwise it won't be changed.

Reply:
It isn't changed, so I see no reason to capture it by reference.

Reply:
I mean it is used in lambda. And you capture some trash stack value.
`curl_easy_strerror(curlcode));`